src/Tests/Microsoft.NET.Restore.Tests/GivenThatWeWantToUseFrameworkRoslyn.cs

```diff
@@ -27,6 +28,8 @@ public void It_restores_Microsoft_Net_Compilers_Toolset_Framework_when_requested
                 TargetFrameworks = "net6.0",
             };
 
+            // Add an explicit version for the test. This will normally come from the installer.
+            project.AdditionalProperties.Add("_NetFrameworkHostedCompilersVersion", "4.7.0-2.23260.7");
```

Comment:
nit: From which feed is that package restored? If it's a dotnetX feed, then we will need to rev the version when we drop that feed.

Reply:
dotnet-tools

Why are we dropping feeds? I would've thought the feed would be permanent even if the particular version may disappear some day.

Reply:
Dotnet-tools is a permanent feed so that's good then. We are dropping dotnetX feeds in subsequent releases, ie dotnet6* feeds shouldn't be used when the stack uses dotnet8* feeds.